maintainers/scripts/common-updater/update-source-version

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env nix-shell
+#! nix-shell --pure -p bash nix coreutils gawk gnused -i bash
+set -e
+
+# FIXME: It's dumb that this is needed.
```

Comment:
Otherwise this happens (on NixOS):
````
error: Nix database directory ‘/nix/var/nix/db’ is not writable: Permission denied
./maintainers/scripts/common-updater/update-source-version: error: Couldn't evaluate 'thunderbird.meta.position' to locate the .nix file!
````